app/src/main/kotlin/com/simplemobiletools/smsmessenger/helpers/Constants.kt

```diff
@@ -30,6 +30,7 @@ const val EXPORT_SMS = "export_sms"
 const val EXPORT_MMS = "export_mms"
 const val EXPORT_MIME_TYPE = "application/json"
 const val EXPORT_FILE_EXT = ".json"
+const val JSON_IMPORT_MIME_TYPE = "application/json"
```

Comment:
we can use this to replace EXPORT_MIME_TYPE as it isnt clear enough

Reply:
Should we name it differently to make it clear it is used for both import and export?

Reply:
lets name it JSON_MIME_TYPE and XML_MIME_TYPE, no need to mention exporting and importing there. Do the same with file extensions please like JSON_FILE_EXTENSION.